docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md

```diff
@@ -30,58 +31,85 @@ Users and Service Accounts can perform a set of predetermined actions along Cons
+| Manage Identities of this Company                                          | `console.company.users.manage`                         |        |          |           |            |                       | ✅            |
+| Edit Project information of all Projects of this Company               | `console.company.project.details.update`               |        |          |           |            | ✅                    | ✅            |
+| Manage Identities of all Projects of this Company                      | `console.company.project.users.manage`                 |        |          |           |            | ✅                    |               |
+| Delete this Company                                                           | `console.company.delete`                               |        |          |           |            |                       | ✅            |
+| Delete all Projects of this Company                                       | `console.company.project.delete`                       |        |          |           |            | ✅                    | ✅            |
+| Manage Providers of this Company                                             | `console.company.providers.manage`                     |        |          |           |            |                       | ✅            |
+| View Company Providers information                                         | `console.company.providers.view`                       | ✅     | ✅       | ✅        | ✅         | ✅                    | ✅            |
+| Manage Clusters for this Company                                              | `console.company.cluster.manage`                       |        |          |           |            |                       | ✅            |
+| View Company Clusters information                                          | `console.company.cluster.view`                         | ✅     | ✅       | ✅        | ✅         | ✅                    | ✅            |
+| Manage Company Project Templates                                           | `console.company.templates.manage`                     |        |          |           |            |                       | ✅            |
+
+Roles can be assigned at both Project and its Runtime Environments levels from the Identities portal of the specific Project Settings Area.
+
+:::info
+Note that into Project Identities portal, only those Identities on which a Company role has previously been assigned are shown.  
+For each of them, therefore, the inherited permissions granted by Company role assignment at Company level will be visible. In this way, it will be possible to identify which additional permissions among those mentioned in he following table can be assigned at Project and Runtime Environments levels. 
```

Comment:
```suggestion
Please note that, inside Project Identities portal, only those Identities on which a Company role has previously been assigned are shown.  
For each of them, therefore, the permissions inherited from the Company role assignment at the Company level will be visible. In this way, it will be possible to identify which additional permissions among those mentioned in the following table can be assigned at Project and Runtime Environments levels. 
```